cluster/log-dump.sh

```diff
@@ -131,7 +131,7 @@ function save-logs() {
       esac
     fi
 
-    if log-dump-ssh "${node_name}" "sudo systemctl status kubelet.service" &> /dev/null; then
+    if log-dump-ssh "${node_name}" "which journalctl" &> /dev/null; then
```

Comment:
Use `command` or `type` instead. Look [here](http://stackoverflow.com/questions/592620/check-if-a-program-exists-from-a-bash-script) for details.